recipes/aodp/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "2.5.0.1" %}
+{% set sha256 = "85b3a1b58cdbbf35ed2666111b5ad8138454c82eb77fbb9348945b83e2a8dab9" %}
+
+package:
+  name: aodp
+  version: {{ version }}
+
+source:
+  url: https://github.com/peterk87/aodp/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - gxx_linux-64 ==7.3.0 # [linux]
```

Comment:
Can you please remove this compiler and also perl/bioperl from the build section.

Reply:
Removed in c05b380. Thanks for the feedback @bgruening !